slushfile.js

```diff
@@ -324,6 +333,8 @@ gulp.task('init', ['checkForUpdates'], function (done) {
   var prompts = [
     {type: 'list', name: 'mlVersion', message: 'MarkLogic version?', choices: ['8','7', '6', '5'], default: 0},
     {type: 'input', name: 'marklogicHost', message: 'MarkLogic Host?', default: 'localhost'},
+    {type: 'input', name: 'marklogicAdminUser', message: 'MarkLogic Admin User?', default: 'admin'},
+    {type: 'input', name: 'marklogicAdminPass', message: 'MarkLogic Admin Password?', default: 'admin'},
```

Comment:
Can you make the default for the admin password blank? Something you can't enter yourself, and works with current serve-local too. It is configured such that you need to call /api/user/login first to initialize session, so default user is not used yet. (It only becomes relevant if you want to disable auth, like I did for github-search..)